plugins/chartjs-adapter-dayjs.js

```diff
@@ -0,0 +1,84 @@
+import { _adapters } from 'chart.js'
+import dayjs from 'dayjs'
+import customParseFormat from 'dayjs/plugin/customParseFormat'
+import 'dayjs/locale/en'
+import 'dayjs/locale/ja'
+import 'dayjs/locale/ko'
+import 'dayjs/locale/pt-br'
+import 'dayjs/locale/th'
+import 'dayjs/locale/vi'
+import 'dayjs/locale/zh-cn'
+import 'dayjs/locale/zh-tw'
+
+const DEFAULT_FORMATS = {
+  datetime: 'MMM D, YYYY, h:mm:ss a',
+  millisecond: 'h:mm:ss.SSS a',
+  second: 'h:mm:ss a',
+  minute: 'h:mm a',
+  hour: 'hA',
+  day: 'MMM D',
+  week: 'll',
+  month: 'MMM YYYY',
+  quarter: '[Q]Q - YYYY',
+  year: 'YYYY'
+}
+
+export function useDayjsAdapter(nuxtI18n) {
```

Comment:
```suggestion
import 'dayjs/locale/zh-tw'
import { NuxtVueI18n } from './nuxt-i18n'

const DEFAULT_FORMATS = {
  datetime: 'MMM D, YYYY, h:mm:ss a',
  millisecond: 'h:mm:ss.SSS a',
  second: 'h:mm:ss a',
  minute: 'h:mm a',
  hour: 'hA',
  day: 'MMM D',
  week: 'll',
  month: 'MMM YYYY',
  quarter: '[Q]Q - YYYY',
  year: 'YYYY'
}

export function useDayjsAdapter(nuxtI18n: NuxtVueI18n) {
```

こんな感じで型定義も明確にしておくと良さそうです

Reply:
https://github.com/tokyo-metropolitan-gov/covid19/blob/25f482fd7fcb684965ffbc84771c770d743d6ed4/plugins/chartjs-adapter-dayjs.js#L1
このadapterが型定義が無く、そこまで変更されるようなファイルでないと判断して、一旦jsで書いてました。
上記の型定義とadapterの型定義を追加してtsにした方がいいですよね？

Reply:
できれば基本は ts にしてビルド通すほうが良いかと！

Reply:
ひとまず、 ts にしました！